torch/nn/quantizable/modules/activation.py

```diff
@@ -0,0 +1,344 @@
+import torch
+from torch import nn
+import torch.nn.functional as nnF
+import torch.nn.quantized as nnq
+
+from torch import Tensor
+from typing import Optional, Tuple
+
+import warnings
+
+class MultiheadAttention(nn.MultiheadAttention):
+    _FLOAT_MODULE = nn.MultiheadAttention
+
+    r"""Allows the model to jointly attend to information
```

Comment:
optional: could we refer to the original docs without copy-paste?  ex: https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/activation.py#L91

Reply:
IMO it would be good for this docblock to explain why this class exists compared to the original MHA class